R/finalize_explanation.R

```diff
@@ -153,3 +168,238 @@ compute_shapley_new <- function(internal, dt_vS) {
 
   return(dt_kshap)
 }
+
+
+
+#' Mean Squared Error of the Contribution Function `v(S)`
+#'
+#' @param internal List with the different parameters, data and functions used internally in the [explain()] function.
+#' @param processed_vS_list List of the processed contribution function estimates.
+#' Output from the \code{shapr:::postprocess_vS_list()} function.
+#' @param p Numeric vector with the predicted responses for the observations which are to be explained.
+#' Output from the \code{shapr:::get_p()} function.
+#' @param exclude_empty_grand_coalitions Boolean. If `TRUE`, we exclude the empty and grand coalitions
+#' when computing the MSEv evaluation criterion. This is reasonable as they are identical for all methods, i.e.,
+#' their contribution function is independent of the used method as they are special cases not effected by
+#' the used method. If `TRUE`, we exclude the empty and grand coalitions.
+#' @param return_as_dt Boolean. If the computed MSEv evaluation criterion is
+#' to be returned as \code{\link[data.table]{data.table}}.
+#' @param use_shapley_weights Boolean. If `TRUE`, then we use the Shapley kernel weights to weight the
+#' different coalitions before taking the mean. If `FALSE`, we use regular mean with uniform weights.
```

Comment:
write (default) in parenthesis for the default. this also goes for the above

Reply:
Added default to the documentation.